core/tests/test_utils.py

```diff
@@ -1317,12 +1317,6 @@ def _create_valid_question_data(self, default_dest_state_name):
         return state
 
 
-if feconf.PLATFORM == 'gae':
-    GenericTestBase = AppEngineTestBase
```

Comment:
@vojtechjelinek: What does `GenericTestBase` now refer to? This class is overridden in `BaseEditorControllerTest` (in editor_test.py) and  `GetHandlerTypeIfExceptionRaisedTest` and `I18nDictsTest` (in base_test.py) (and a number of other places) but there seem to be no reference to it in test_utils.py. Is this intended? (Sorry for basic question! I am just trying to figure this out).